core/test/Operation/Boolean/TestEquality.hs

```diff
@@ -0,0 +1,30 @@
+{-# LANGUAGE OverloadedLists #-}
+
+module Operation.Boolean.TestEquality where
+
+import JsonLogic.Json
+import JsonLogic.Pure.Evaluator
+import Test.Tasty (TestTree, testGroup)
+import Test.Tasty.HUnit (testCase)
+import qualified Test.Tasty.HUnit as U
+
+-- See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Equality
+equalityUnitTests :: TestTree
+equalityUnitTests =
+  testGroup
+    "Read json unit tests"
```

Comment:
I think this has not yet been changed to the correct thing after copying.

Reply:
```suggestion
    "Equality unit tests"
```